File: programming_basics/exercises/01_exercise_first_steps_in_coding/09_fish_tank.py
length_cm = int(input())
width_cm = int(input())
height_cm = int(input())
capacity_percent = float(input())
empty_volume_tank_cm = length_cm * width_cm * height_cm
# 1 liter is equal to 1 dm^3
empty_volume_tank_liters = empty_volume_tank_cm * 0.001  # volume_tank_cm / 1000
capacity_water = capacity_percent / 100
needed_water = empty_volume_tank_liters * (1 - capacity_water)
print(needed_water)

File: 05_JavaScriptBasics/02_Exercises/01_FirstStepsInCoding/09_fishTank.js
function aquarium(input){
    // 1 liter = 1 dm^3
    let lengthCm =  Number(input[0])
    let widthCm = Number(input[1])
    let heigthCm = Number(input[2])
    let percent = Number(input[3]) / 100

    let volumeAquarium = (lengthCm * widthCm * heigthCm) / 1000 
    let availableSpace = volumeAquarium * (1 - percent)

    console.log(availableSpace)
}
